lib/output.js

```diff
@@ -726,6 +727,14 @@ function tile (options) {
     } else if (is.defined(options.layout) && options.layout === 'google') {
       this.options.tileSkipBlanks = 5;
     }
+    // Center image in tile
+    if (is.defined(options.centre) || is.defined(options.center)) {
+      if (is.bool(options.centre) || is.bool(options.center)) {
+        this.options.centre = options.centre || options.center;
```

Comment:
Please can you use `this._setBooleanOption()` to set this boolean property, preferably with a `tile` prefix for consistency, i.e. `tileCentre`. You may also want to set it to a default value of `false` when `this.options` is created in the constructor.